core/server/worker/src/test/java/alluxio/worker/block/DefaultBlockWorkerTest.java

```diff
@@ -384,6 +392,78 @@ public void updatePinList() {
     verify(mBlockStore).updatePinnedInodes(pinnedInodes);
   }
 
+  void setPersistenceConf() {
+    // write permission is required,
+    // so the default WORKER_PERSISTENCE_INFO_PATH is set to temporary folder
+    ServerConfiguration.set(PropertyKey.WORKER_CLUSTERID_PATH,
+        mTestFolder.getRoot().getAbsolutePath());
+  }
+
+  @Test
+  public void handlePreRegisterCommandACK_REGISTER()
+      throws IOException, BlockDoesNotExistException, InvalidWorkerStateException {
+    String newClusterId = UUID.randomUUID().toString();
+    GetWorkerIdPResponse response = GetWorkerIdPResponse.newBuilder()
+        .setPreRegisterCommandType(PreRegisterCommandType.ACK_REGISTER)
+        .setClusterId(newClusterId).setWorkerId(mWorkerId1).build();
+
+    // Only workerId will be set, ClusterId will not be Set, So it will get EMPTY_CLUSTER_ID
+    mBlockWorker.handlePreRegisterInfo(response);
+    assertEquals(IdUtils.EMPTY_CLUSTER_ID,
+        mBlockWorker.getOrDefaultClusterIdFromDB(IdUtils.EMPTY_CLUSTER_ID).get());
+    assertEquals(1L, (long) mBlockWorker.getWorkerId().get());
+  }
+
+  @Test
+  public void handlePreRegisterCommandREGISTER_PERSIST_CLUSTERID()
+      throws IOException, BlockDoesNotExistException, InvalidWorkerStateException {
+    String newClusterId = UUID.randomUUID().toString();
+    GetWorkerIdPResponse response = GetWorkerIdPResponse.newBuilder()
+        .setPreRegisterCommandType(PreRegisterCommandType.REGISTER_PERSIST_CLUSTERID)
+        .setClusterId(newClusterId).setWorkerId(mWorkerId1).build();
+
+    // the new cluster ID will be persisted
+    mBlockWorker.handlePreRegisterInfo(response);
+    assertEquals(newClusterId,
+        mBlockWorker.getOrDefaultClusterIdFromDB(IdUtils.EMPTY_CLUSTER_ID).get());
+    assertEquals(mWorkerId1, (long) mBlockWorker.getWorkerId().get());
+  }
+
+  @Test
+  public void handlePreRegisterCommandREGISTER_CLEAN_BLOCKS()
```

Comment:
I think you can just use camel case here instead of using underscores. So `handleRegisterCleanBlocks()`

Reply:
done